-meeting-template.md

```diff
@@ -1,5 +1,7 @@
 ---
-private: false
+breaks:  false # See: https://github.com/hackmdio/codimd/issues/40#issuecomment-172927690
+private: false # See: https://github.com/hyphacoop/hyphacoop-chatbot#archive
+
```

Comment:
Curious why this extra newline?

Reply:
heh nothing gets by you ;) github code editor was formatting it oddly when it was directly above `---`, thinking it was a title.

will remove

Reply:
```suggestion
```